gtsam/slam/slam.i

```diff
@@ -322,6 +322,15 @@ template <T = {gtsam::Point2, gtsam::Rot2, gtsam::Pose2, gtsam::Point3,
 virtual class KarcherMeanFactor : gtsam::NonlinearFactor {
   KarcherMeanFactor(const gtsam::KeyVector& keys);
 };
+  
+class Rot3Vector {
+  Rot3Vector();
+
+  // structure specific methods
+  gtsam::Rot3 at(size_t i) const;
+  void push_back(const gtsam::Rot3& R);
+};
+const gtsam::Rot3 FindKarcherMean<gtsam::Rot3>(const Rot3Vector& rotations);
```

Comment:
no need for `const ` in the return type: `gtsam::Rot3 FindKarcherMean<gtsam::Rot3>(const Rot3Vector& rotations);`